src/js/fetches/movies.js

```diff
@@ -23,9 +23,9 @@ export const checkPopular = async (page = 1) => {
 
 //  2.    ------ function fetch - Movie checker - by title ------ 
 // movieTitle is a .value from header input
-export const checkMovie = async (movieTitle) => {
+export const getMovie = async (movieTitle) => {
     try {
-      const response = await fetch(apiUrl+apiKey+'&query='+ movieTitle+'&page='+ page);
+      const response = await fetch(apiUrl+`?api_key=` + apiKey + '&query=' + movieTitle+'&page=' + page);
```

Comment:
Nadal brak spacji przy operatorze "+" przed ?api_key. Edytuj to :)

Reply:
zmienione